### DISCUSSION TITLE
Linked in profile

### DISCUSSION THREAD

**Discussion by shiinedev** [dugsiiyeinc/dhaqsoPay#102] [General]
<img width="662" alt="team" src="https://github.com/user-attachments/assets/8d130ca3-aa60-4eb7-bd31-f6a925f47269" />

 @dugsiiyeinc/dhaqsopay-contributors  asxaabta qof walbo oo meshan ku jiro linked in profile linkigiisa haso diro si loogo xiro

**Comment by Nasrah-muse** [comment]
@shiinedev  waakan kayga eng https://www.linkedin.com/in/nasro-muuse-0a4374343/

**Reply by iftiiin** [reply]
https://www.linkedin.com/in/iftin-awil-7b80861a6?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app
@shiinedev waa kan eng

**Comment by faaraxcabdulaahi** [comment]
> <img alt="team" width="662" src="https://private-user-images.githubusercontent.com/171542930/398131507-8d130ca3-aa60-4eb7-bd31-f6a925f47269.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUxNDI1NzksIm5iZiI6MTczNTE0MjI3OSwicGF0aCI6Ii8xNzE1NDI5MzAvMzk4MTMxNTA3LThkMTMwY2EzLWFhNjAtNGViNy1iZDMxLWY2YTkyNWY0NzI2OS5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyNVQxNTU3NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kOTFmNzgwYTZjNGU5NTgwYjliNTQ3NjhhZGM0MzJhYjUzOGRjZTFmZGQ2ZDA5MmRhN2RiMGY1ZjY3OTMwZWNkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._2VbdExDX3_1I2aTCywMmKZs0gupI5S12cvJVzrHfhc">
> @dugsiiyeinc/dhaqsopay-contributors asxaabta qof walbo oo meshan ku jiro linked in profile linkigiisa haso diro si loogo xiro

https://www.linkedin.com/in/faarax-cabdulaahi-576500282?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=ios_app
